src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private Object[] keys = new Object[MAX_SIZE];
+    private Object[] values = new Object[MAX_SIZE];
```

Comment:
Use the parameterized types
All non-constant fields should be initialized in the constructor
```suggestion
    private K [] keys;
    private V [] values;
```